src/test/java/net/sf/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -112,6 +112,12 @@ public void testMonthChecks() {
     }
 
     @Test
+    public void testBibtexkeyChecks() {
+        assertCorrect(createContext("bibtexkey", "Knuth2014"));
```

Comment:
As we now have a class for each checker, we should also create a test class for each checker separately.
But I would think that is content for a new PR